Makefile

```diff
@@ -102,7 +102,7 @@ requirements.txt:
 
 $(build.proto): $(src.proto)
 	@$(call i, Generating python source for protobuf)
-	protoc -I $(src.proto.dir) --python_out=$(build.proto.dir) $(src.proto)
+	protoc -I $(src.proto.dir) --python_out=$(build.proto.dir) --experimental_allow_proto3_optional $(src.proto)
```

Comment:
Add this to the Java build, or drop it if unneeded.

Reply:
I don't think we need the optional tag on the protobuf fields, they can be empty in general. The 'optional' may be useful when consuming code requires a default value, but in general fields can be empty and we handle that.